pkgs/tools/misc/nvimpager/default.nix

```diff
@@ -29,15 +29,19 @@ stdenv.mkDerivation rec {
     '';
 
   doCheck = true;
-  checkInputs = [ lua51Packages.busted util-linux neovim ];
+  checkInputs = [ lua51Packages.busted unixtools.script neovim ];
   checkPhase = ''
     runHook preCheck
-    script -c "busted --lpath './?.lua' test"
+  '' + (
+  if !stdenv.isDarwin then ''
+    ${unixtools.script}/bin/script -c "busted --lpath './?.lua' test"
+  '' else ''
+    ${unixtools.script}/bin/script "busted --lpath './?.lua' test"
```

Comment:
`script` on Darwin requires passing a file to also pass a command:
```suggestion
    ${unixtools.script}/bin/script /dev/null busted --lpath './?.lua' test
```

However, the tests fail:
```
Failure → test/nvimpager_spec.lua @ 237
cat mode with modeline ignores mode lines in git log diffs #osx_pending
test/nvimpager_spec.lua:241: Expected objects to be equal.
Passed in:
(string) 'Error detected while processing modelines:
line   19:
E518: Unknown option: foobar=errorcommit 3d032f9e546fc8a1e987bbbef54e8dab1ceaf4bd
Author: Lucas Hoffmann <lucc@posteo.de>
Date:   Mon Aug 20 23:37:04 2018 +0200

    This is based on the log entry of 3d032f9e546fc8a1e987bbbef54e8dab1ceaf4bd
    but was modiefied for the tests.

    The last line is turned into a valid modeline even with the diff prefix.
    If it is interpreted it should result in an error message.

diff --git a/test/fixtures/help.txt b/test/fixtures/help.txt
index ad4baf6..ab64439 100644
--- a/test/fixtures/help.txt
+++ b/test/fixtures/help.txt
@@ -1,3 +1,3 @@
-vim: filetype=help
 *conceal-test* normal
 conceal-test
+ vim: foobar=error
'
Expected:
(string) 'commit 3d032f9e546fc8a1e987bbbef54e8dab1ceaf4bd
Author: Lucas Hoffmann <lucc@posteo.de>
Date:   Mon Aug 20 23:37:04 2018 +0200

    This is based on the log entry of 3d032f9e546fc8a1e987bbbef54e8dab1ceaf4bd
    but was modiefied for the tests.

    The last line is turned into a valid modeline even with the diff prefix.
    If it is interpreted it should result in an error message.

diff --git a/test/fixtures/help.txt b/test/fixtures/help.txt
index ad4baf6..ab64439 100644
--- a/test/fixtures/help.txt
+++ b/test/fixtures/help.txt
@@ -1,3 +1,3 @@
-vim: filetype=help
 *conceal-test* normal
 conceal-test
+ vim: foobar=error
'

Failure → test/nvimpager_spec.lua @ 722
parent detection detects git correctly #osx_pending
test/nvimpager_spec.lua:725: Expected objects to be equal.
Passed in:
(string) 'nil'
Expected:
(string) 'git'

Failure → test/nvimpager_spec.lua @ 728
parent detection detects man correctly #osx_pending
test/nvimpager_spec.lua:731: Expected objects to be equal.
Passed in:
(string) 'nil'
Expected:
(string) 'man'

Failure → test/nvimpager_spec.lua @ 741
parent detection handles man #osx_pending
test/nvimpager_spec.lua:745: Expected objects to be equal.
Passed in:
(string) 'TEST(0)                                                                TEST(0)

SYNOPSIS
       exit

AUTHORS
       Author.

                                     date                              TEST(0)
'
Expected:
(string) 'TEST(0)                                                                TEST(0)

SYNOPSIS
       exit

AUTHORS
       Author.

                                     date                              TEST(0)
'
```